src/python/grpcio_health_checking/grpc_health/v1/_async.py

```diff
@@ -83,7 +87,7 @@ async def _set(
         status: _health_pb2.HealthCheckResponse.ServingStatus,
     ) -> None:
         if service in self._server_watchers:
-            condition = self._server_watchers.get(service)
+            condition = self._server_watchers[service]
```

Comment:
Why this change?

Reply:
.get can give None so pytype was flaging it but we know this would never be None using this notation silences that error by pytype.